.github/workflows/publish-auxiliary-packages.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Auxiliary Packages
+on:
+  # NOTE: avoid using the manual execution unless is the only way to fix some issue,
+  # rather retry failed jobs in case of flakes. The manual execution can potentially
+  # cause the tags to point to a different commit that the one used to publish
+  # the packages
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+        startsWith(github.event.head_commit.message, 'chore(release): bump auxiliary package versions')
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Create Github App Token
+      uses: mongodb-js/devtools-shared/actions/setup-bot-token@main
+      id: app-token
+      with:
+        app-id: ${{ vars.DEVTOOLS_BOT_APP_ID }}
+        private-key: ${{ secrets.DEVTOOLS_BOT_PRIVATE_KEY }}
+    
+    - uses: actions/checkout@v4
+      with:
+        # don't checkout a detatched HEAD
+        ref: ${{ github.head_ref }}
```

Comment:
Need to pass in the token from the previous step.

Reply:
Will add, also duplicated from https://github.com/mongodb-js/compass/blob/main/.github/workflows/publish-packages.yaml